packages/css-color-parser/src/serialize/p3.ts

```diff
@@ -9,7 +9,7 @@ import { toPrecision } from './to-precision';
 import { XYZ_D50_to_P3_Gamut } from '../gamut-mapping/p3';
 
 export function serializeP3(color: ColorData, gamutMapping = true): FunctionNode {
-	color.channels = convertPowerlessComponentsToMissingComponents(color.channels, color.colorNotation);
+	color.channels = convertPowerlessComponentsToZeroValuesForDisplay(color.channels, color.colorNotation);
```

Comment:
"for display" is taken from the specification.

In our context it means that this color will be serialized to a string, and not further processed.

This makes it possible to give true white in `rgb` for `lab(100 0 10)`.
Which would otherwise become something non-white due to gamut mapping.